### PR TITLE
Marketplace: handle community edition a bit differently

### PR DESCRIPTION
**Pull request by gregaubert**

### CODE REVIEW

File: server/sonar-web/src/main/js/apps/marketplace/components/EditionBoxBadge.tsx [hunk 1]
```diff
@@ -28,22 +28,34 @@ interface Props {
 }
 
 export default function EditionBoxBadge({ editionKey, status }: Props) {
```
Comment (SonarTech) [inline server/sonar-web/src/main/js/apps/marketplace/components/EditionBoxBadge.tsx]: CRITICAL The function EditionBoxBadge has a cyclomatic complexity of 11 which is higher than the threshold of 10 [rule](https://next.sonarqube.com/sonarqube/coding_rules#rule_key=typescript%3AS1541)

**Comment by SonarTech** [comment]
SonarQube analysis reported 1 issue
* ![CRITICAL][CRITICAL] 1 critical

Watch the comments in this conversation to review them.

[CRITICAL]: https://sonarsource.github.io/sonar-github/severity-critical.png 'Severity: CRITICAL'

**Comment by gregaubert** [comment]
@stas-vilchik would you review the last commit please ? :pray:

File: server/sonar-web/src/main/js/app/components/nav/settings/SettingsEditionsNotif.tsx [hunk 2]
```diff
@@ -81,12 +82,21 @@ export default class SettingsEditionsNotif extends React.PureComponent<Props, St
...
+          <span className="spacer-left">
+            <FormattedMessage
+              defaultMessage={translate('marketplace.see_url_to_enable_cluster')}
+              id="marketplace.see_url_to_enable_cluster"
+              values={{
+                url: (
+                  <a
+                    href="https://redirect.sonarsource.com/doc/data-center-edition.html"
+                    target="_blank">
+                    Data Center Edition
```
Comment (stas-vilchik) [inline server/sonar-web/src/main/js/app/components/nav/settings/SettingsEditionsNotif.tsx]: l10n

File: server/sonar-web/src/main/js/apps/marketplace/components/LicenseEditionSet.tsx [hunk 3]
```diff
@@ -149,12 +150,21 @@ export default class LicenseEditionSet extends React.PureComponent<Props, State>
...
+          <span className="little-spacer-left">
+            <FormattedMessage
+              defaultMessage={translate('marketplace.how_to_setup_cluster_url')}
+              id="marketplace.how_to_setup_cluster_url"
+              values={{
+                url: (
+                  <a
+                    href="https://redirect.sonarsource.com/doc/data-center-edition.html"
+                    target="_blank">
+                    Data Center Edition
```
Comment (stas-vilchik) [inline server/sonar-web/src/main/js/apps/marketplace/components/LicenseEditionSet.tsx]: l10n

File: sonar-core/src/main/resources/org/sonar/l10n/core.properties [hunk 4]
```diff
@@ -2110,13 +2110,14 @@ marketplace.release_notes=Release Notes
...
+marketplace.status.MANUAL_IN_PROGRESS=Commercial Edition can't automatically be installed because of internet access issues. Please manually install the package.
 marketplace.status_x.AUTOMATIC_IN_PROGRESS=Installing your new {0}... Please wait...
 marketplace.status_x.AUTOMATIC_READY={0} successfully installed. Please restart the server to activate your new edition.
 marketplace.status_x.UNINSTALL_IN_PROGRESS=Successfully downgraded to {0}. Please restart the server to remove the features.
-marketplace.status_x.MANUAL_IN_PROGRESS={0} can't automatically be installed because of internet access issues. Please manually install the package in your SonarQube's plugins folder.
+marketplace.status_x.MANUAL_IN_PROGRESS={0} can't automatically be installed because of internet access issues. Please manually install the package.
 marketplace.how_to_install=How to install it?
-marketplace.how_to_configure_cluster=How to configure a cluster?
+marketplace.see_url_to_enable_cluster=See {url} to enable clustered operation.
```
Comment (stas-vilchik) [inline sonar-core/src/main/resources/org/sonar/l10n/core.properties]: See {url} documentation to set up a cluster.

File: sonar-core/src/main/resources/org/sonar/l10n/core.properties [hunk 5]
```diff
@@ -2110,13 +2110,14 @@ marketplace.release_notes=Release Notes
...
+marketplace.how_to_setup_cluster_url=Further setup and configuration are required to enable clustered operation. See {url}.
```
Comment (stas-vilchik) [inline sonar-core/src/main/resources/org/sonar/l10n/core.properties]: ... to get a cluster. See {url} documentation.

**Pull request merged**